src/EditorFeatures/CSharpTest/CodeActions/IntroduceVariable/IntroduceVariableTests.cs

```diff
@@ -4073,7 +4073,7 @@ void M()
     {
         int a = 1;
         int {|Rename:y1|} = C.y;
-        var t = (a, x: y1);
+        var t = (a: a, x: y1);
```

Comment:
Yes.  That would be very unideal.  Do we ahve any sort of "always simplify" annotation that indicates that we should ignore user preferences and def simplify if possible?  Tagging @DustinCampbell